inference-engine/src/mkldnn_plugin/nodes/mkldnn_conv_node.cpp

```diff
@@ -129,6 +108,30 @@ void MKLDNNConvolutionNode::getSupportedDescriptors() {
     if (!descs.empty())
         return;
 
+    if (getAlgorithm() == ConvolutionCommon) {
```

Comment:
Seems like this change is not actual anymore?

Reply:
removed